lib/cdn-site-hosting/cdn-site-hosting-props.ts

```diff
@@ -17,4 +17,5 @@ export interface CommonCdnSiteHostingProps {
   sourcesWithDeploymentOptions?: SourcesWithDeploymentOptions[];
   websiteErrorDocument?: string;
```

Comment:
Question: do we need any custom treatment for when a separate `websiteErrorDocument` is provided for an SPA? Not sure of the use case for this either